<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atom</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/atomoIcon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="control">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>531</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/Xbox One.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Adelante">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>271</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/EstadoCarro/estadoInicial.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="giroDerecha">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/EstadoCarro/giroDerecha_burned.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="giroIzquierda">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>321</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/EstadoCarro/giroIzquierda_burned.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="EstadoConectado">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/rec.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="EstadoDesconectado">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/rec (1).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BuscandoConexion">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/rec (2).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="press_a">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>611</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="press_x">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>595</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="press_y">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>580</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="press_b">
    <property name="geometry">
     <rect>
      <x>671</x>
      <y>596</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="press_leftJ">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>586</y>
      <width>75</width>
      <height>48</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="press_rightJ">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>623</y>
      <width>75</width>
      <height>48</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Conectar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/boton-encendido-simbolo-universal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rightJ">
    <property name="geometry">
     <rect>
      <x>563</x>
      <y>599</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/flecha-derecha (1).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leftJ">
    <property name="geometry">
     <rect>
      <x>518</x>
      <y>599</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/izquierda.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="upJ">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>577</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/arriba.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="downJ">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>619</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/abajo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rightJ_2">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>635</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/flecha-derecha (1).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="upJ_2">
    <property name="geometry">
     <rect>
      <x>633</x>
      <y>614</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/arriba.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leftJ_2">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>635</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/izquierda.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="downJ_2">
    <property name="geometry">
     <rect>
      <x>633</x>
      <y>655</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/abajo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cuadritos">
    <property name="geometry">
     <rect>
      <x>583</x>
      <y>599</y>
      <width>27</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lineas">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>599</y>
      <width>27</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/destello.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>637</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/flecha-derecha (1).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="left">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>638</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/izquierda.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="up">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>615</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/arriba.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="down">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>660</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/abajo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="offcanon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/dispararListo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oncanon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/dispararListo2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="canon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/canon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LB">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>41</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/menos.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RB">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>41</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/menos.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RT">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>41</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/menos.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LT">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>41</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/interfaceControl/menos.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="manual">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/autonomo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="offmanual">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/dispararListo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="onmanual">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/dispararListo2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="carga">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/carga.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="offcarga">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/dispararListo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oncarga">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/canon/dispararListo2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Fondo">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-30</y>
      <width>891</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/fondo3.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="desconectar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Fondos/images.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Fondo</zorder>
   <zorder>logo</zorder>
   <zorder>control</zorder>
   <zorder>Adelante</zorder>
   <zorder>giroDerecha</zorder>
   <zorder>giroIzquierda</zorder>
   <zorder>EstadoConectado</zorder>
   <zorder>EstadoDesconectado</zorder>
   <zorder>BuscandoConexion</zorder>
   <zorder>press_a</zorder>
   <zorder>press_x</zorder>
   <zorder>press_y</zorder>
   <zorder>press_b</zorder>
   <zorder>press_leftJ</zorder>
   <zorder>press_rightJ</zorder>
   <zorder>Conectar</zorder>
   <zorder>rightJ</zorder>
   <zorder>leftJ</zorder>
   <zorder>upJ</zorder>
   <zorder>downJ</zorder>
   <zorder>rightJ_2</zorder>
   <zorder>upJ_2</zorder>
   <zorder>leftJ_2</zorder>
   <zorder>downJ_2</zorder>
   <zorder>cuadritos</zorder>
   <zorder>lineas</zorder>
   <zorder>right</zorder>
   <zorder>left</zorder>
   <zorder>up</zorder>
   <zorder>down</zorder>
   <zorder>offcanon</zorder>
   <zorder>oncanon</zorder>
   <zorder>canon</zorder>
   <zorder>LB</zorder>
   <zorder>RB</zorder>
   <zorder>RT</zorder>
   <zorder>LT</zorder>
   <zorder>manual</zorder>
   <zorder>offmanual</zorder>
   <zorder>onmanual</zorder>
   <zorder>carga</zorder>
   <zorder>offcarga</zorder>
   <zorder>oncarga</zorder>
   <zorder>desconectar</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="AtomImg/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
